tools/scripts/add-credits-to-project.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+set -eu
+
+SITE="https://credits.apps.staging.hpc.cam.ac.uk"
+
+
+# Set up some variables
+CONTENT_TYPE="Content-Type: application/json"
+
+TOKEN="CHANGE THIS AFTER REDEPLOY" 
+
+AUTH_HEADER="Authorization: Bearer $TOKEN"
+
+# rcp-azimuth-cloud-portal-dev 
+PROJECT_ID="79b2c7925276436091fa9301c4b05fd2"
+
+# 3. Add an account
+echo "Adding an account:"
+ACCOUNT_ID=$(curl -s -X POST -H "$AUTH_HEADER" -H "$CONTENT_TYPE" -d \
+    '{
+        "name": "Tes1452 Account",
+        "email": "test251@account.com"
+    }' \
+    $SITE/account/ | jq -r '.url')
+echo "Account URL: $ACCOUNT_ID"
+
+PROVIDER_ID="$SITE/resource_provider/1/"
+
+echo "Adding a resource provider account:"
```

Comment:
So this script fails when the account already exists, I think we need to make the script idempotent, ideally.